docs/application/dotnet/guides/nui/resources.md

```diff
@@ -38,8 +67,8 @@ Visual creation automatically determines whether the image is N_PATCH, SVG, or G
 | `CropToMask`         | Flag to determine if main content image should crop to match mask size. |
 | `FittingMode`        | By default the image will shrink to fit the DesiredHeight and DesiredWidth, other modes available. |
```

Comment:
Please update as below:

By default, the image will shrink to fit the DesiredHeight and DesiredWidth, other modes available.